Disable expected test failures.

TCPStoreTest is too slow to run.

Reference: https://github.com/pytorch/pytorch/issues/13746
Reference: https://github.com/pytorch/pytorch/issues/21873

Index: pytorch-1.4.0_p0/torch/lib/c10d/test/CMakeLists.txt
===================================================================
--- pytorch-1.4.0_p0.orig/torch/lib/c10d/test/CMakeLists.txt
+++ pytorch-1.4.0_p0/torch/lib/c10d/test/CMakeLists.txt
@@ -14,7 +14,6 @@ function(c10d_add_test test_src)
 endfunction()
 
 c10d_add_test(FileStoreTest.cpp c10d)
-c10d_add_test(TCPStoreTest.cpp c10d)
 
 if(USE_CUDA)
   if(USE_C10D_GLOO)
Index: pytorch-1.4.0_p0/caffe2/share/contrib/depthwise/CMakeLists.txt
===================================================================
--- pytorch-1.4.0_p0.orig/caffe2/share/contrib/depthwise/CMakeLists.txt
+++ pytorch-1.4.0_p0/caffe2/share/contrib/depthwise/CMakeLists.txt
@@ -6,4 +6,3 @@ set(Caffe2_CONTRIB_DEPTHWISE3x3_TEST_CPU
 )
 
 set(Caffe2_CPU_SRCS ${Caffe2_CPU_SRCS} ${Caffe2_CONTRIB_DEPTHWISE3x3_CPU_SRC} PARENT_SCOPE)
-set(Caffe2_CPU_TEST_SRCS ${Caffe2_CPU_TEST_SRCS} ${Caffe2_CONTRIB_DEPTHWISE3x3_TEST_CPU_SRC} PARENT_SCOPE)
